ingenialink/dictionary.py

```diff
@@ -1085,6 +1078,28 @@ def __read_bitfields(
 
         return None
 
+    def __add_register(self, register: Register, axis: int) -> None:
+        """Adds a register to register list.
+
+        Args:
+            register: register to add.
+            axis: register's axis.
+
+        Raises:
+            ValueError: if register identifier is not provided.
+        """
+        if axis not in self._registers:
+            self._registers[axis] = {}
+            if axis == 0:
+                self.subnodes[axis] = SubnodeType.COMMUNICATION
+            else:
+                if axis != 1:
+                    logger.warning(f"Found {axis=}, will treat it as a Motion axis.")
```

Comment:
for multiaxis xdf v2 I think this must be implicit. Throwing warning is warning of something the user should not care I think...